### PR TITLE
navigation between login and register screens

### LINKED ISSUE #22: As a user, I want to be able to navigate between the login and regsiter screen, so that if I already have an/no account, I can navigate to the corresponding screen
This depends on the navigation actions

### PR DESCRIPTION
**Pull request by octogradiste**
This will close #22.

### CODE REVIEW

**Review by unglazedstamp — CHANGES_REQUESTED** [review]
Very clean code !
just some minor things to change.

File: app/src/androidTest/java/com/github/swent/echo/ui/navigation/NavigationTest.kt [hunk 1]
```diff
@@ -1,69 +1,59 @@
...
 import org.junit.Rule
 import org.junit.Test
-import org.junit.runner.RunWith
 
-@RunWith(AndroidJUnit4::class)
+@HiltAndroidTest
 class NavigationTest {
 
-    @get:Rule val composeTestRule = createComposeRule()
+    @get:Rule(order = 0) val hiltRule = HiltAndroidRule(this)
```
[the remark this answers is not in the document]
Comment (octogradiste) [reply app/src/androidTest/java/com/github/swent/echo/ui/navigation/NavigationTest.kt:18]: I tried to run the tests without, and it doesn't work... My guess is that we need to specify the order, even if we don't actually use the `hiltRule`.

**Comment by octogradiste** [comment]
Thank you so much for taking the time to review my code :)

**Review by unglazedstamp — APPROVED** [review]
All good, thank you for the improvements !

**Pull request merged**